docs/storage.md

```diff
@@ -461,40 +461,66 @@ For the full discussion about Ceph in XCP-ng, see this forum thread: <https://xc
 
 ## ISO SR
 
-You might wonder: how to upload an ISO? Unlike other solutions, you need to create a dedicated "space" for those, a specific ISO SR. To create one, it's pretty easy.
+You might wonder: how to upload an ISO? Unlike other solutions, you need to create a dedicated "space" for those, a specific ISO SR. To create a ISO SR, you have 2 possibilities:
+- Shared: A shared ISO SR is on a VM or in your dedicated storage server. It's accessible with a IP address, like 192.168.1.100
+- Local (not recommended): Local ISO SR is a directory created directly on the dom0. It's only accessible on the pool where the dom0 ith the directory on it.
 
-In [Xen Orchestra](management.md#xen-orchestra) go into "New/Storage" and select "ISO SR":
+Create a Shared ISO SR:
 
-![](https://xcp-ng.org/assets/img/screenshots/createSRISO.png)
+First, you need to create the NFS or SMB Share. The creation depends on the device you want to use to store your ISO, but here a small tutorial to mount a NFS share on Debian 11:
 
-Usually, you can use a NFS or SMB share. But if you don't have any of those, you can rely on a local ISO SR that you create yourself:
+```
+fdisk /dev/xvdb #create a new partition
+mkfs.ext4 -b 4096 /dev/xvdb1 # format the partition to ext4
+apt install nfs-kernel-server
+mkdir /ISOs
+mount /dev/xvdb1 /ISOs #mount the disks to /ISOs
+emacs /etc/fstab
+  /dev/xvdb	  /ISOs	  /ext4	  defaults	0	1
+emacs /etc/exports
+  /ISOs 192.168.X.0/24(rw,sync,no_subtree_check,no_root_squash)
+exportfs -arv
+```
```

Comment:
I'm not convinced that we should provide these step by step instructions. As I said on the forum, I would prefer pointers to well established guides on the internet.

By the way, the style of our docs makes the example stand out too much compared to the rest of the section that is more important in my eyes

![image](https://user-images.githubusercontent.com/451043/186887596-99b2a006-ff1e-4f7f-ab4e-e1231e59bf65.png)

@olivierlambert any opinion?

Reply:
Indeed, doing an NFS share is likely outside the scope of XCP-ng doc. Linking to a popular guide would be better IMHO.